www-plugins/passff-host/passff-host-1.0.2.ebuild

```diff
@@ -1,20 +1,18 @@
 # Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-PYTHON_COMPAT=( python3_4 python3_5 python3_6 )
+PYTHON_COMPAT=( python3_5 python3_6 )
```

Comment:
python 3.7 maybe?

Reply:
Yup thanks for the notice.
I tested it and https://docs.python.org/3/whatsnew/3.7.html says that the only breaking change is two keywords that aren’t used in passff-host.